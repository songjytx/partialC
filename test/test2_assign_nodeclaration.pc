void main(){
	int a;
	printi(a);
    float b;
    printf(b);
    bool c;
    if(c){
    	prints('correct');
    }else{
    	prints('wrong');
    }
    string d;
    prints(d);

    arr int e[2];
    e = [2,3];
    printi(e[0]);
}