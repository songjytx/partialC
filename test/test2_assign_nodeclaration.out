0
0.000000
'correct'

2
